content/en/contribute/code/core/deploy-on-eks.md

```diff
@@ -81,7 +81,12 @@ After you have created a ticket per "Request permission" above, you should get a
    ```shell
    kubectl config use-context arn:aws:eks:eu-west-2:720541322708:cluster/dev-cht-eks
    ```
-3. Add new `values.yaml` file - you can [copy this one](https://github.com/medic/medic-infrastructure/blob/master/terraform/aws/dev/cht-projects/alpha-dev-cht-deploy-values.yaml) and just change the `alpha-dev` values to `USERNAME-dev` and change the value in `ingress` -> `annotations` -> `certificate` to the latest updated value(request the SRE team for the latest value). Remember where you created this file - you'll need it every time you use the `cht-deploy` script.
+3. Create a new `values.yaml` file by [copying this one](https://github.com/medic/medic-infrastructure/blob/master/terraform/aws/dev/cht-projects/alpha-dev-cht-deploy-values.yaml). Be sure to update these values after you create it: 
+   * `alpha-dev` values to `USERNAME-dev` 
+   * Update `certificate`  to the latest value from SRE - currently it's `arn:aws:iam::720541322708:server-certif    icate/2023-wildcard-dev-medicmobile-org-chain`
```

Comment:
```suggestion
   * Update `certificate`  to the latest value from SRE - currently it's `arn:aws:iam::720541322708:server-certif    icate/2024-wildcard-dev-medicmobile-org-chain`
```
With https://github.com/medic/helm-charts/pull/5 this might become optional